servicetalk-dns-discovery-netty/src/test/java/io/servicetalk/dns/discovery/netty/DnsServiceDiscovererBuilderProviderTest.java

```diff
@@ -42,7 +53,13 @@ void appliesBuilderProvider() {
         public DnsServiceDiscovererBuilder newBuilder(final String id, final DnsServiceDiscovererBuilder builder) {
             buildCounter.incrementAndGet();
             buildId.set(id);
-            return new DelegatingDnsServiceDiscovererBuilder(builder);
+            return new DelegatingDnsServiceDiscovererBuilder(builder) {
+                @Override
+                public DnsServiceDiscovererBuilder ttlJitter(final Duration ttlJitter) {
+                    ttlJitterIntercept.set(ttlJitter.toMillis());
+                    return super.ttlJitter(ttlJitter);
```

Comment:
While this doesn't matter much in this particular case, we had issues in the past when users used `super` and the super impl changed. We always recommend users to use `delegate()` instead for well-defined behavior. Consider changing it to `delegate()` here too, bcz we sometimes reference tests as examples.